js/coinsbit.js

```diff
@@ -913,6 +918,16 @@ module.exports = class coinsbit extends Exchange {
         } else {
             orderSize = amountGeneric;
         }
+        // only after we have filled/unfilled amounts calculated, we can calculate status after that
+        const sameFillAsAmount = orderSize === filledSize;
+        if (isFromFetchOpenOrders) {
+            status = 'open';
+        } else if (isFromFetchClosedOrders) {
+            status = sameFillAsAmount ? 'closed' : 'cancelled';
```

Comment:
The correct unified spelling for it throughout the lib is `canceled`, not `cancelled`.